packages/main/test/samples/Dialog.sample.html

```diff
@@ -158,4 +158,43 @@ <h3>Draggable and Resizable Dialog</h3>
 <ui5-dialog draggable resizable></ui5-dialog>
 	</xmp></pre>
 </section>
+
+<section>
+	<h3>Fiori styled footer</h3>
```

Comment:
We should use the full name - SAP Fiori. Also, headings are with title case: 
SAP Fiori Styled Footer